src/Info.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Section = styled.section`
+  max-width: 740px;
+  margin: auto;
+  margin-bottom: 130px;
+`;
+
+const Title = styled.p`
+  font-size: 24px;
+  font-family: 'Bitter', serif;
+`;
+
+const Article1 = styled.article`
+  margin-top: 130px;
+`;
+
+const List = styled.ul`
+  color: #93918f;
+  line-height: 27px;
+  margin-top: -15px;
+  margin-left: -35px;
+  list-style: none;
+`;
+
+const ListItem = styled.li`
+  &:before {
+    content: '.';
+    position: relative;
+    top: -4px;
+    right: 8px;
+    font-size: 24px;
+  }
+`;
+
+const Article2 = styled.article`
```

Comment:
An alternative to having two separate `Article` components would be to have a single one with `margin-top: 105px` and set `padding-top: 25px` to the `Section` component.